fdbclient/DatabaseContext.h

```diff
@@ -364,7 +384,9 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 	                                   std::unique_ptr<SpecialKeyRangeReadImpl> &&impl);
 
 	static bool debugUseTags;
-	static const std::vector<std::string> debugTransactionTagChoices; 
+	static const std::vector<std::string> debugTransactionTagChoices;
+	private:
+		std::map<Key, Reference<WatchMetadata>> watchMap;
```

Comment:
I think it's fine to make `watchMap` public in `DatabaseContext`, since `DatabaseContext` should only be accessible to internal code. Looks like a bunch of these fields used to be private but aren't anymore (https://github.com/apple/foundationdb/blob/master/fdbclient/DatabaseContext.h#L215). Maybe someone with more knowledge can comment here.